src/core/src/mapping.rs

```diff
@@ -112,9 +120,11 @@ impl<R: Resources> Raw<R> {
         sync(&mut inner);
 
         Writer {
-            len: len,
+            // len: len,
+            // inner: inner,
+            // phantom: PhantomData,
```

Comment:
Reimplementation of the Writer from RWer
```
/// Mapping writer.
/// Currently is not possible to make write-only slice so while it is technically possible
/// to read from Writer, it will lead to an undefined behavior. Please do not read from it.
pub type Writer<'a, R, T> = RWer<'a, R, T>;
```